skills/nuxt-ui/references/layouts/editor.md

```
@@ -2,26 +2,26 @@
 
 Build a rich text editor with toolbars, slash commands, mentions, and drag-and-drop.
 
+## When to use
+
+- Note-taking apps, CMS editors
+- Collaborative editing interfaces
+- Any rich text editing need (supports JSON, HTML, and Markdown)
+
 ## Component tree
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)